### DISCUSSION TITLE
Referring to classes in Mirth Conn

### DISCUSSION THREAD

**Discussion by donmc07** [nextgenhealthcare/connect#5991] [General]
I don't have much java coding experience. So apologies in advance for any stupid questions.
I need to use Java class SerialClob in my Mirth connect channel. I just don't know how to refer to it. Any help with some sample code is really appreciated.
I have the following article about this class from Oracle. Just don't know how to refer to it in Mirth connect. Neet to use the function getCharacterStream() in this class.
https://docs.oracle.com/javase/8/docs/api/javax/sql/rowset/serial/SerialClob.html

**Comment by pacmano1** [comment]
What is the use case here?   Curious.  Is this in the context of a database insert?   Maybe `Packages.java.sql.Clob` like in `paramList.add(new Packages.java.sql.Clob(clobData))`. (not tested)
